### DISCUSSION TITLE
force sync with the NVD database

### DISCUSSION THREAD

**Discussion by ellipse2v** [DependencyTrack/dependency-track#1315] [Q&A] [answered]
hi

could you tell me if it's possible to force the synchronisation with the nvd ?
thanks
Regards
Sylvain

**Comment by stevespringett** [comment, marked as answer]
You can delete the ./nist directory in the DT data directory and restart DT. That will force synchronization.

See also: https://docs.dependencytrack.org/getting-started/data-directory/

**Comment by ellipse2v** [comment]
Ok
Thanks


Le lun. 3 janv. 2022 à 15:47, Steve Springett ***@***.***> a
écrit :
